frontend/apps/picasso/apollo/apolloGraphql.ts

```diff
@@ -1,11 +1,6 @@
-import {
-  ApolloClient,
-  InMemoryCache,
-  ApolloProvider,
-  gql,
-} from "@apollo/client";
+import { ApolloClient, InMemoryCache } from "@apollo/client";
 
 export const client = new ApolloClient({
   uri: process.env.SUBSQUID_URL,
-  cache: new InMemoryCache(),
+  cache: new InMemoryCache()
```

Comment:
@easteregg i think this might enable cache for all queries, do you want all data to be cached?

Reply:
Yeah, defining policies on a per query basis is best. Although we can do other cache mechanism, but I wanted to light on this to not rely on "invalid data" when it comes to fetching queries.